dockerfiles/Dockerfile

```diff
@@ -0,0 +1,81 @@
+FROM openjdk:8
```

Comment:
how big is the difference between jdk and jre base image i mean if they have any?